runtime/jcl/common/mgmtthread.c

```diff
@@ -1276,7 +1277,10 @@ freeThreadInfos(J9VMThread *currentThread, ThreadInfo *allinfo, UDATA allinfolen
 {
 	PORT_ACCESS_FROM_VMC(currentThread);
 	SynchronizerInfo *sinfo, *next;
-	UDATA i;
+	UDATA i, j;
+	J9JavaVM *vm = currentThread->javaVM;
+	J9InternalVMFunctions *vmfns = vm->internalVMFunctions;
+
 
```

Comment:
While you're here, please fix the existing declarations (one variable per declaration, with an initializer).
Please don't add the extra blank line (we don't need two or more together).
```c
	SynchronizerInfo *sinfo = NULL;
	SynchronizerInfo *next = NULL;
	UDATA i = 0;
	UDATA j = 0;
```